packages/input-manager/src/devices/device.ts

```diff
@@ -0,0 +1,44 @@
+import EventEmitter from 'events'
```

Comment:
I like using `eventemitter3` as an alternative which allows for easy strong typing.